mongodb-advanced/index.js

```diff
@@ -0,0 +1,50 @@
+const async = require('async');
+const bedrock = require('bedrock');
+const path = require('path');
+const config = bedrock.config;
+const database = require('bedrock-mongodb');
+
+require('bedrock-express');
+require('bedrock-server');
+require('bedrock-views');
+
+// load config
+require('./config');
+
+const rootPath = path.join(__dirname);
+config.views.system.packages.push({
+  path: path.join(rootPath, 'components'),
+  manifest: path.join(rootPath, 'package.json')
+});
+
+// open some collections once the database is ready
+bedrock.events.on('bedrock-mongodb.ready', callback => async.auto({
+  openCollections: callback =>
+    database.openCollections(['mongodb_advanced'], callback)
+}, err => callback(err)));
+
+bedrock.events.on('bedrock-express.configure.routes', addRoutes);
+
+function addRoutes(app) {
+  app.get('/people', (req, res) => {
+    database.collections.mongodb_advanced.find({}).toArray(function(err, docs) {
```

Comment:
use arrow function

Reply:
also, the app.get API provides a third option: next, which you should use as an error handler here:
```
if (err) {
  return next(err)
}
```

Reply:
Not sure I implemented this one correctly so definitely look at this one